api/HTMLOutputElement.json

```diff
@@ -48,6 +48,55 @@
           "deprecated": false
         }
       },
+      "HTMLOutputElement": {
+        "__compat": {
+          "spec_url": "https://html.spec.whatwg.org/multipage/form-elements.html#htmloutputelement",
+          "description": "<code>HTMLOutputElement()</code> constructor",
+          "support": {
+            "chrome": {
+              "version_added": false
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": false
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": false
+            },
+            "opera_android": {
+              "version_added": false
+            },
+            "safari": {
+              "version_added": "15"
```

Comment:
Curious that this constructor works but many others don't. But this seems correct!